### ISSUE TITLE
how to get one point without repeat

### ISSUE THREAD

**Issue by MaJun0007** [ghettovoice/vuelayers#109]
hey mate , Thanks for vuelayers , i use vl-layer-image get one map but when i use vl-geom-point it while be repeat x ,How to get one point ? and how to use `< vl-source-xyz **:url="Array"**>`

**Comment by ghettovoice** [comment]
Hi @MaJun0007 !
About multiple `vl-geom-point`: if you render all of them with `v-for` you use `ref` from Vue:
```html
...
// this.$refs.points - now is array of vl-geom-point component instances
<vl-geom-point ref="points" v-for="point in points" :coordinates="point.coordinates"></vl-geom-point>
...
```

Why you need an array as url for `vl-source-xyz`? I'm asking about this because if you need define multiple tile subdomains you can use url temlate placeholders `{[a-z]-[a-z]}` or `{[0-9]-[0-9]}` for this.
For example : `http://{a-f}.example.com/tiles/{z}/{x}/{y}.png`.

**Comment by ghettovoice** [comment]
For the most advanced usage you can use mixin [`xyzSource`](https://github.com/ghettovoice/vuelayers/blob/master/src/mixin/xyz-source.js) to build your own XYZ source component. This mixin encapsulates all features that you need, also an additional tokens for url templates. Take a look on mapbox source for exmaple https://github.com/ghettovoice/vuelayers/blob/master/src/component/mapbox-source/source.vue

**Comment by MaJun0007** [comment]
Thank you for your reply! Maybe I didn't say it clearly enough .
i want get one point not multiple. but when i use vl-geom-point it will be auto repeat x 
this is my code
```
<vl-feature id="point">
          <vl-geom-point :coordinates="[50, 50]"></vl-geom-point>
</vl-feature>
```
![snip20181126_18](https://user-images.githubusercontent.com/23500089/48990561-e9bae580-f169-11e8-8e60-a20375af38a5.png)

**Comment by ghettovoice** [comment]
Can you show me full component template where you add map?

**Comment by ghettovoice** [comment]
This can be related with wrapX feature. May be you should disable it for layers

**Comment by MaJun0007** [comment]
```
<template>
  <div id="app">
    <vl-map :load-tiles-while-animating="true" :load-tiles-while-interacting="true"
             data-projection="EPSG:4326">
      <vl-view :zoom.sync="zoom" :center.sync="center" :rotation.sync="rotation">

      </vl-view>
       <vl-layer-image id="jz">
        <vl-source-image-static
          :url="map"
          :size="imageSize"
          :extent="imageExtent">
        </vl-source-image-static>
      </vl-layer-image>
      <vl-feature id="point">
          <vl-geom-point :coordinates="[50, 50]"></vl-geom-point>
      </vl-feature>
      <!-- <vl-layer-tile id="xyz"
          :extent="imageExtent">
        <vl-source-xyz
          :extent="imageExtent"
          :attributions="attribution"
          :url="map"
          :layer-name="layerName"
          :matrix-set="matrixSet"
          :format="format"
          :style-name="styleName" ></vl-source-xyz>
      </vl-layer-tile> -->
    </vl-map>
    <div class="zomm">
      Zoom: {{ zoom }}<br>
      Center: {{ center }}<br>
      Rotation: {{ rotation }}<br>
    </div>
  </div>
</template>

<script>
let imageSizeH = 480
let imageSizeW = 611
let x = imageSizeW * 200000
let y = imageSizeH * 200000
let imageExtent = [-x / 2, -y / 2, x / 2, y / 2]
export default {
  name: 'App',
  data () {
    return {
      imageSize: [imageSizeW, imageSizeH],
      imageExtent: imageExtent,
      zoom: 0,
      center: [480 / 2, 611 / 2],
      rotation: 0,
      maps: [
        require('./images/map/world_0.png'),
        require('./images/map/world_1.png')
      ],
      map: require('./images/map/world_0.png'),
      layerName: '0',
      matrixSet: 'EPSG:3857',
      format: 'image/jpg',
      styleName: 'default',
      attribution: 'Tile'
    }
  },
  watch: {
    zoom (value) {
      this.map = this.maps[value]
    }
  }
}
</script>

<style>
html,body, #app {
  height: 100%;
  width: 100%;
  overflow: hidden;
}
#app {
  position: relative;
  background-color: #000;
}
.zomm {
  position: absolute;
  top: 0;
  left: 0;
}
</style>

```

**Comment by MaJun0007** [comment]
and i want to konw how to show <vl-overlay> toggle by mouseover not select thank you

**Comment by ghettovoice** [comment]
When you add `vl-feature` just inside `vl-map`, that feature will be appended to the default map overlay.
I released the new version v0.11.1 where `vl-map` now alsp  has a property `wrap-x` to control wrapping around X axis on that overlay. Each `vl-source-*` component also has the same property that is enabled by default. 

But for case I suggest you to render features inside special vector layer, not on the default overlay, because select interaction also uses an internal overlay for styling selected features an two overlay can interfere with each other when selection occures.

I prepare extended example with your use case, take a look here https://jsfiddle.net/489fgwm6/7/

**Comment by MaJun0007** [comment]
OK, I will try  today, Thanks again.
